client/src/index.css

```diff
@@ -70,3 +71,16 @@ body {
   padding: 5px 10px;
   background-color: lightskyblue;
 }
+
+a[aria-current="true"] {
+  color: #000;
+  font-weight: bold;
+  cursor: pointer;
```

Comment:
Aside from relying on [`aria-current`](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/current/) (which is used for accessibility purposes to indicate that an element is currently active/selected), you can also utilize the [`activeClassName`](https://reacttraining.com/react-router/web/api/NavLink/activeclassname-string) attribute in a `<NavLink>` tag.